src/models/searchDocument.js

```diff
@@ -0,0 +1,17 @@
+/* 
+* @Author: Mike Reich
+* @Date:   2016-02-21 07:04:27
+* @Last Modified 2016-02-21
+*/
+
+'use strict';
+
+import {BaseModel} from '@nxus/storage'
+
+export default BaseModel.extend({
+  identity: 'searchdocument',
+  connection: 'searcher',
+  attributes: {
+    
```

Comment:
Model attribute is set in the createHandler, though I suppose it could be defined as a discrete attribute here.